packages/trader/src/Stores/Modules/Trading/trade-store.ts

```diff
@@ -1616,8 +1741,4 @@ export default class TradeStore extends BaseStore {
             }
         }
     }
-
-    setStakeBoundary(type, min_stake, max_stake) {
```

Comment:
@henry-deriv please keep this version of `setStakeBoundary`,  please only add types to params that I added in TS migration PR for trade-store: https://github.com/henry-deriv/deriv-app/pull/5/files#diff-f2458dff61c55e54d7921b962d102cfcbfe5911d22e4fea81cd2ad76857e7b88

Reply:
removed

Reply:
done

Reply:
@henry-deriv `if (min_stake && max_stake)` is still missing, please add it🙏🏻

Reply:
is it necessary?

I think `If (min_stake && max_stake)` was added to get rid of typescript error

Reply:
i think changing 

stake_boundary to `Partial<TStakeBoundary>` got rid of the typescript error

Reply:
thank you @henry-deriv :)